tests/test_get_request_body_schema.py

```diff
@@ -0,0 +1,36 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+from pydantic import BaseModel
+
+app = FastAPI()
+
+
+class Product(BaseModel):
+    name: str
+    description: str = None
+    price: float
+
+
+@app.get("/product")
+async def create_item(product: Product):
+    pass
```

Comment:
i think this is where the coverage is missing so try adding `# pragma: nocover` here

Reply:
hey thanks @rkbeatss . It worked!

Reply:
Thanks for the help @rkbeatss ! :nerd_face: